src/commcare_cloud/ansible/letsencrypt_cert.yml

```diff
@@ -123,3 +123,14 @@
             - running `commcare-cloud <env> ansible-playbook deploy_proxy.yml`
             - optionally, copying the important files from `proxy` `/etc/pki/tls/`
               to an encrypted file in the environment settings.
+
+
+    - name: Add certbot cron
+      cron:
+        name: "Certbot Renew"
+        job: "test -x /usr/bin/certbot && perl -e 'sleep int(rand(3600))' && certbot -q renew"
```

Comment:
The looks like it checks for the existence of `/usr/bin/certbot`, waits for a randomized amount of time that is between 0 seconds and 1 hour, and then renews the cert. The randomized wait period?

Reply:
It was set by the certbot . Certbot servers have some limit on the number of requests you can make in a particular duration. I believe it is for that.

Reply:
Oh I see, they probably would rather that everyone space out their requests as opposed to having a billion requests be made to their servers right on the hour. Clever